src/views/Home.vue

```diff
@@ -15,7 +17,14 @@
       <div class="tab-content card-body">
         <div class="tab-pane fade show active" id="projects">
           <div class="card-text">
-            <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#add-project-modal"><i class="fas fa-plus"></i> Create Project</button>
```

Comment:
This line now has to be warped around, is this what we want?
Does this look better?